.github/workflows/demo.yml

```
@@ -0,0 +1,135 @@
+      - name: Commit and push changes
+        uses: EndBug/add-and-commit@v9
+        with:
+          author_name: "GitHub Actions"
+          author_email: "github-actions[bot]@users.noreply.github.com"
+          message: |
+            chore: update screengrabs for ${{ needs.prepare.outputs.version }}
+          new_branch: "chore/update-build-screengrabs-for-${{ needs.prepare.outputs.version }}"
+          push: --force-with-lease
+
+      - name: Create or update PR
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)